src/masternodes/mn_rpc.cpp

```diff
@@ -743,6 +747,27 @@ UniValue setgovheight(const JSONRPCRequest& request) {
             throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, res.msg);
         }
         varStream << name << *gv;
+
+        if (name == "ATTRIBUTES") {
+            const auto attributes = std::dynamic_pointer_cast<ATTRIBUTES>(gv);
+            if (!attributes) {
+                throw JSONRPCError(RPC_INVALID_REQUEST, "Failed to convert Gov var to attributes");
+            }
+
+            LOCK(cs_main);
+            const auto attrMap = attributes->GetAttributesMap();
+            for (const auto& [key, value] : attrMap) {
+                if (const auto attrV0 = std::get_if<CDataStructureV0>(&key)) {
+                    DCT_ID tokenID{attrV0->typeId};
+                    if (attrV0->type == AttributeTypes::Consortium &&
+                        (attrV0->typeId == 0 ||
+                         !pcustomcsview->GetToken(tokenID)->IsDAT() ||
```

Comment:
Do not use `->IsDAT()` unless you know a valid object has been returned or we will segfault.